src/monster.cpp

```diff
@@ -277,15 +288,16 @@ void monster::try_upgrade(bool pin_time) {
         return;
     }
 
-    const int current_day = to_days<int>( calendar::time_of_cataclysm - calendar::turn );
-
+    //const int current_day = to_days<int>( calendar::time_of_cataclysm - calendar::turn );
```

Comment:
Why is there commented out code?

Reply:
This is what I suspect was preventing evolutions, which is why I reversed it in the next line.  The only reason I didn't delete it was that I figured there was a small chance that the <> check later in the code could have been reversed, giving a similar effect.

Reply:
If it's wrong code, remove it (removed code can still be accessed via git). If it's correct, keep it.

Commenting it out is just confusing for everyone.

> This is what I suspect was preventing evolutions

Sounds about right. The old code will yield negative number of days.